pkg/types/tuf/v0.0.1/tuf_key_v0_0_1_schema.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I don't see this file referenced anywhere, is it used?

Reply:
An artifact, nope -- public keys are just held in the same manifest format as the artifact